statsd.yaml

```diff
@@ -26,11 +28,49 @@ pipeline:
         cp -r $d ${{targets.destdir}}/usr/src/app
       done
 
+  - runs: |
+      cd ${{targets.contextdir}}/usr/src/app
+      # optional includes hashring, a dep for proxy.js
+      npm install --omit=dev
+
   - runs: |
       # Set graphite hostname to "graphite"
       sed -i 's/graphite.example.com/graphite/' exampleConfig.js
       mv exampleConfig.js ${{targets.destdir}}/usr/src/app/config.js
 
+test:
+  environment:
+    contents:
+      packages:
+        - nodejs-22
```

Comment:
Looks great. Just one minor nitpick - in our tests here, we pin to nodejs 22, but above we install hashring with the latest version of nodejs. The two should align

Reply:
thanks for the review.  I dropped the '22' here.